app_dart/lib/src/model/appengine/task.dart

```diff
@@ -45,6 +45,24 @@ class Task extends Model {
     id = key?.id;
   }
 
+  // Represents a LUCI build.
+  Task.chromebot(
+    this.commitKey,
+    this.createTimestamp,
+    this.name,
+    this.isFlaky,
+  )   : assert(createTimestamp != null),
+        assert(name != null),
+        assert(isFlaky != null),
+        timeoutInMinutes = 0,
```

Comment:
does 0 means unlimited?

Reply:
"The timeout of the task, or zero if the task has no timeout"